bindings/Sofa/tests/Simulation/Node.py

```diff
@@ -141,8 +141,6 @@ def test_getItem(self):
             object2 = node2.addObject("MechanicalObject", name="object2")
 
             # All those are allowed syntaxes:
-            self.assertEqual(root[""].name, root.name)
```

Comment:
Why did you remove these ?

Reply:
Because when I look at the `__getitem__` binding functions, an empty path (`[""]`) is only supported with objects, and not with nodes:

```
py::object __getitem__(Node& self, const std::string& s) {
(...)
    if(stringlist.empty())
        throw py::value_error("Invalid path provided");
(...)
}
```

Object binding do not have this exception.